src/content/api/programs.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Programs
+description: Program model and related endpoints
+nav:
+  path: API
+  order: 17
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A Program can be used to provide a versatile foundation for diverse organisational requirements.
+Comprising three distinct types - `loyalty`, `acceptance` and `retail`.
+
+- **Loyalty Programs** enable organisations to set up Promotions and Reward their customers.
+A Consumer can connect with a Loyalty Program to participate in Promotions and earn Rewards.
+
+- **Acceptance Programs** allow organisations to create a new [Asset Type](/api/asset-types) on Centrapay’s platform.
+Acceptance Programs allow Merchants to connect with Asset Issuers to accept their Asset Type in-store.
+
+- **Retail Programs** allow Merchants to connect with POS/Terminal providers to activate Centrapay payments on their devices.
+
+## Program Model
+
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier of the Program.
+  </Property>
+
+  <Property name="name" type="string">
+    The display name of the Program.
+  </Property>
+
+  <Property name="type" type="string">
+    The type of a Program. Current supported value is `loyalty`. 'retail' and 'acceptance' at a later date.
```

Comment:
We shouldn't mention future stuff, so maybe: The type of functionality contained in the program. Valid values: “loyalty”.